react/bingo/src/bingo/BingoApp.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import BingoGame from './Game'
+import './BingoApp.css';
+
+class BingoApp extends React.Component {
+
+  constructor (props){
+    super(props);
+    this.state = {
+      challengeWord : '',
+      isGameOn : false
+    }
+  }
+
+  handleSubmit(){
+    this.setState ({
+      challengeWord : this.state.challengeWord,
+      isGameOn : true
+    });
+    playBingo(this.state.challengeWord);
+  }
+
+  refreshPage () {
+    window.location.reload();
+    this.setState ({
+      isGameOn : false,
+      challengeWord : ''})
+  }
+
+  render() {
+    return (
+          <div>
```

Comment:
If the <div>'s are just wrappers and not adding to the layout of the compoent, then they can be replaced with the following tag, <React.Fragment>your content</React.Fragment>

Reply:
I have added the styles at the div level. After changing to the React.Fragment, the className attribute wasn't there. Hence need to check further on this comment. Will keep this unresolved.

Reply:
As told in the review comment, if the div is just for the sake of wrapping the react element/s returned in an opening and closing tag, then <React.Fragment> can be used as it servers the purpose of the same.